src/Aws/Lambda/Runtime.hs

```diff
@@ -68,6 +69,9 @@ invokeWithCallback callback event context = do
                       , executionUuid = ""  -- DirectCall doesnt use UUID
                       }
   result <- callback lambdaOptions
+  -- Flush output handlers to insure output goes into CloudWatch logs
```

Comment:
Could you extract this into a `flushOutput` function with a doc comment please?

Reply:
Fixed.